ui/Screen/Project/Source.svelte

```diff
@@ -98,6 +100,8 @@
       );
     } catch (error) {
       notification.error(`Checkout failed: ${error.message}`, true);
+    } finally {
+      screen.unLock();
```

Comment:
Should we make sure that there is an `unlock` for the destruction of this component as well?

Reply:
Probably doesn't make sense, because there is no automatic call to navigate anywhere after the backend request finishes and since the user can't click on anything, there is no way to close it. We'll have to rely on that `try ... catch ... finally`.